frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -490,113 +501,135 @@ const PropertyListView: React.FC = () => {
+          pageCount={pageCount}
+          onSortChange={(column: string, direction: SortDirection) => {
+            if (!!direction) {
+              setSorting({ ...sorting, [column]: direction });
+            } else {
+              const data: any = { ...sorting };
+              delete data[column];
+              setSorting(data);
             }
           }}
-        >
-          <Form>
-            <DirtyRowsTracker setDirtyRows={setDirtyRows} />
-            <Table<IProperty>
-              name="propertiesTable"
-              lockPageSize={true}
-              columns={columns}
-              data={data || []}
-              loading={data === undefined}
-              filterable
-              sort={sorting}
-              pageIndex={pageIndex}
-              onRequestData={handleRequestData}
-              pageCount={pageCount}
-              onSortChange={(column: string, direction: SortDirection) => {
-                if (!!direction) {
-                  setSorting({ ...sorting, [column]: direction });
-                } else {
-                  const data: any = { ...sorting };
-                  delete data[column];
-                  setSorting(data);
-                }
-              }}
-              canRowExpand={(val: any) => {
-                if (val.values.propertyTypeId === 0) {
-                  return true;
-                } else {
-                  return false;
-                }
-              }}
-              detailsPanel={{
-                render: val => {
-                  if (expandData[val.id]) {
-                    return <Buildings hideHeaders={true} data={expandData[val.id]} />;
+          canRowExpand={(val: any) => {
+            if (val.values.propertyTypeId === 0) {
+              return true;
+            } else {
+              return false;
+            }
+          }}
+          detailsPanel={{
+            render: val => {
+              if (expandData[val.id]) {
+                return <Buildings hideHeaders={true} data={expandData[val.id]} />;
+              }
+            },
+            icons: {
+              open: <FaFolderOpen color="black" size={20} />,
+              closed: <FaFolder color="black" size={20} />,
+            },
+            checkExpanded: (row, state) => !!state.find(x => checkExpanded(x, row)),
+            onExpand: loadBuildings,
+            getRowId: row => row.id,
+          }}
+          filter={appliedFilter}
+          onFilterChange={values => {
+            setFilter({ ...filter, ...values });
+          }}
+          renderBodyComponent={({ body }) => (
+            <Formik
+              innerRef={tableFormRef as any}
+              initialValues={{ properties: data || [] }}
+              enableReinitialize
+              onSubmit={async values => {
+                let nextProperties = [...values.properties];
+                const changedRows = dirtyRows.map(change => {
+                  const data = { ...values.properties![change.rowId] };
```

Comment:
this works properly with paging, right?

Reply:
Only the current page is part of the form, yes is works with paging